README.md

```diff
@@ -1,142 +1,88 @@
-balena CLI
-=========
+# balena CLI
 
 > The official balena CLI tool.
 
 [![npm version](https://badge.fury.io/js/balena-cli.svg)](http://badge.fury.io/js/balena-cli)
 [![dependencies](https://david-dm.org/balena-io/balena-cli.svg)](https://david-dm.org/balena-io/balena-cli)
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/balena-io/chat)
 
-Getting Started
----------------
+## About
```

Comment:
As currently coded, four sections are parsed and extracted from README.md to be part of the docs web page:
* `About` (at the top, _with its title deleted_)
* `Installation` (effectively just a sentence with link to INSTALL.md)
* `Getting Started`
* `Support, FAQ and troubleshooting`